learn4haskell.cabal

```
@@ -22,7 +22,7 @@ source-repository head
   location:            https://github.com/kowainik/learn4haskell.git
 
 common common-options
-  build-depends:       base ^>= 4.14.0.0
+  build-depends:       base ^>= 4.13.0.0
```

In order to tests to pass, you can not restrict it having the upper version, that is needed for our CI system
```suggestion
  build-depends:       base >= 4.13.0.0 && < 4.15
```

Do you have any particular reason for this change? Does any of our instruction didn't work for you?

---

Yeah, cabal doesn't build with 4.14 version, so I had to use 4.13 in order to conduct chapter tests. I am using ghc version 8.8.4 globally. (I had installed it through ghcup)

---

Oh I see. Yes, indeed, 8.8.4 requires `4.13` 🙂 We are recommending to use the latest `8.10` in course, that is why we put these restrictions 👍🏼 Glad that is working for you!